docs/DataWorkflows/DataCycle.md

```diff
@@ -0,0 +1,671 @@
+
+Currently, the best way to use Globus is either through your local disk or on PMACs (recommended). We're still awaiting CUBIC authorization. The general docs for globus are located [here](https://docs.globus.org/how-to/), but for posterity, here are the best instructions:
+
+On a local disk:
+
+1. Log in to Globus with your UPenn organization account -- [https://docs.globus.org/how-to/get-started/](https://docs.globus.org/how-to/get-started/) -- and try out the tutorial for sharing between two test endpoints on Globus' system
+2. Download and install [Globus Connect Personal](https://www.globus.org/globus-connect-personal); this service will manage the endpoint on your local machine
+3. Download and install the [CLI](https://docs.globus.org/cli/) with pip -- remember to use conda environments! This service will allow you to manage the Globus session when it's running
+4. [Login with the CLI](https://docs.globus.org/cli/quickstart/) and transfer your data either through the CLI commands or by visiting the file manager (which you saw in step 1). If someone has shared a Globus endpoint with your account, you'll have access to it in "Endpoints".
+
+On PMACs:
+
+0. Make sure you have access to the PULSE Secure VPN -- [remote.pmacs.upenn.edu](remote.pmacs.upenn.edu)
+
+1. Log in to PMACs' dedicated node for Globus functionality:
+
+```
+# first ssh into bblsub for network access
+ssh -y <username>@bblsub
+
+# then from there, log onto the globus node
+ssh -y <username>@sciglobus
+```
+
+2. Globus Connect Personal should be available. As above, use it to initialize an endpoint on a directory of your choice on PMACs. Specifically, you should run it as below so that it opens a GUI for logging in with an auto-generated token:
+
+```
+# this command will return a URL you can open in any browser and a token you can use to sign in
+globusconnect -start &
+```
+
+3. Using a new or existing conda environment (see [here](https://pennlinc.github.io/docs/pmacs#logging-in-to-pmacs-lpc) for how to activate conda on PMACs), install the [CLI](https://docs.globus.org/cli/) using `pip` and login with `globus login`.
+
+4. Visit [https://docs.globus.org/how-to/get-started/](https://docs.globus.org/how-to/get-started/) to access the File Manager, as in the Local Disk instructions, to start transferring data.
+
+
+# Curating BIDS Datasets
+
+
+BIDS curation can be a frustrating process. This guide describes best practices for
+the curation process using a local filesystem. We divide this process into multiple
+stages.
```

Comment:
```suggestion
BIDS curation can be a frustrating process. 
This guide describes best practices for the curation process using a local filesystem. 
We divide this process into multiple stages.
```

I find version control on markdown files easier if I split lines based on sentences and commas. No need to adopt this if you don't like it.